src/style.css

```diff
@@ -0,0 +1,42 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+}
+
+header {
+  margin-top: 4rem;
+}
+
+h1 {
+  text-align: center;
+  margin: 1rem 0;
+}
+
+section {
+  margin: 0 auto;
+  width: 50%;
+}
+
+.subtitle-display {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  margin-bottom: 20px;
+}
+
+.subtitle-input {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0;
+}
+
+input {
+  margin-bottom: 8px;
+}
```

Comment:
- [ ] Please as it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md) for this project, " `Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:` ", please kindly update the structure of your page to match the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). It is a required skill for a front-end dev to stick to the design as much as possible. See screenshots below  for more clarifications. 

#### The wireframe

![wireframeChris](https://user-images.githubusercontent.com/65852621/175096754-4b3da2f0-8de7-40fc-88c7-b621fe07ddd2.png)


#### Your implementation
![implementationChris](https://user-images.githubusercontent.com/65852621/175096864-b2143c5c-441a-41d6-a52e-aab7e48a7986.png)



Please kindly fix that.

Reply:
Hello @Mihndim2020 
The scores should be added using javascript so don't know why my work is not validated.
As it is the styling is of little importance. Just using the HTML to create the wireframe is what is required the styling is to be done later in the week.

Reply:
I honestly think the work is based on how well you set up your project using gitflow and webpack not the styling

Reply:
@Christabelle143 well done 👍 implementing the requested changes.